modules/rosa-aws-provisioned.adoc

```diff
@@ -131,3 +131,11 @@ AWS security groups provide security at the protocol and port access level; they
 |`19531`
 
 |===
+
+[id="rosa-security-groups-custom_{context}"]
+=== Additional custom security groups
+When you create a cluster using an existing non-managed VPC, you can add additional custom security groups during cluster creation. Custom security groups are subject to the following limitations:
+
+* You must create the custom security groups in AWS before you create the cluster. For more information, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-security-groups.html[Amazon EC2 security groups for Linux instances].
+* You must associate the custom security groups with the VPC that the cluster will be installed into. Your custom security groups cannot be associated with another VPC.
```

Comment:
This sentence ends in a preposition

Reply:
Per [ISG](https://www.ibm.com/docs/en/ibm-style?topic=grammar-prepositions) you are actually allowed to end in a preposition

> Use a preposition at the end of a sentence to avoid an awkward or stilted construction.